src/styles/blocks/calendar.scss

```
@@ -0,0 +1,46 @@
+.calendar {
+  display: flex;
+  flex-flow: row wrap;
+  gap: $gap;
+  width: $calender-width;
+  margin: 0 auto;
+  padding: 30px $calendar-padding;
+
+  &__day {
+    display: flex;
+    justify-content: center;
+    align-items: center;
+    @include square($day-size);
+    background-color: $day-background-color;
+    border: 1px solid black;
+    transition: all 0.5s ;
```

Don;t do like that, it's to hard for the browser
```suggestion
    transition: background-color 0.5s, ..... ;
```